### PR TITLE
docs: add vite-plugin-html-config plugin

### PR DESCRIPTION
**Pull request by ahwgs**
> Please make sure all the requirements are satisfied, otherwise the PR could be closed without further notice.

## Checklist

- [x] Title as described.
- [x] Make sure you put things in the right category.
- [x] The description of your item should be a sentence with less than 24 words.
- [x] Omit unnecessary words already provided in the context (e.g. `for Vite`, `a Vite plugin`).
- [x] Always add your items to the end of a list.

### Plugins/Tools

- [x] The plugin/tool is working with **Vite 2.0 and onward**.
- [x] The project is Open Source.
- [x] The project follows the [Vite Plugins Conventions](https://vitejs.dev/guide/api-plugin.html#conventions).
- [x] The repo should be at least 30 days old.
- [x] The documentation is in English.
- [x] The project is active and maintained.
- [x] The project accepts contributions.
- [x] Not a commercial product.

### Starter Templates

- [x] The starter template is working with **Vite 2.0 and onward**.
- [x] The documentation is in English.
- [x] The starter template most provide enough instructions / documentation about how to start up and what's is included.
- [x] A screenshot or a live demo should be included.
- [x] The repo should be at least 30 days old.
- [x] Should be differentiable from the existing starter templates. 

### Apps/Websites

- [x] The website is available without errors and load in a reasonable amount of time.
- [x] The website must be Open Source and showing it's using Vite intensively.
- [x] The website is original and not too simple. For that reason, blogs and simple landing pages are rejected.

### CODE REVIEW

**Comment by ahwgs** [comment]
Due to my problem, I closed the previous pr and reopened one

**Review by deleonio — COMMENTED** [review]
What is a real use case of this plugin?

The App artifacts should be decoupled from Index.html oder CMS

**Comment by ahwgs** [comment]
> What is a real use case of this plugin?
> 
> The App artifacts should be decoupled from Index.html oder CMS

https://github.com/ahwgs/vite-plugin-html-config#readme 
I added new instructions,Why is this a CMS?

File: README.md [hunk 1]
```diff
@@ -113,6 +113,7 @@ Use the "Table on Contents" menu on the top-left corner to explore the list.
 - [vite-plugin-html](https://github.com/anncwb/vite-plugin-html) - Plugin to minimize and use ejs template syntax in `index.html`.
 - [vite-plugin-ts-nameof](https://github.com/Shinigami92/vite-plugin-ts-nameof) - Ability to resolve [nameof](https://github.com/dsherret/ts-nameof) in TypeScript.
 - [vite-plugin-handlebars](https://github.com/alexlafroscia/vite-plugin-handlebars) - Process HTML files with Handlebars.
+- [vite-plugin-html-config](https://github.com/ahwgs/vite-plugin-html-config) - Html tag configuration, such as script link style meta favicon and more.
```
Comment (antfu) [inline README.md:116]:
```suggestion
- [vite-plugin-html-config](https://github.com/ahwgs/vite-plugin-html-config) - HTML tag configuration, such as script link style meta favicon and more.
```

**Pull request merged**